src/styles/_catalog.scss

```diff
@@ -1,22 +1,23 @@
-.catalog {
-  display: flex;
-  flex-wrap: wrap;
+$width: 200px;
 
-  max-width: 944px;
+.catalog {
+  display: grid;
   margin: 0 auto;
```

Comment:
Bem block shouldn't have positioning rules at all.

Also, you don't need this for centering content in your BEM block. (use grid css for that)